frontend/packages/dev-console/src/components/pipelines/pipeline-form/SecretsList.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import * as _ from 'lodash';
+import { ResourceLink, Firehose, FirehoseResult } from '@console/internal/components/utils';
+import { SecretModel } from '@console/internal/models';
+import { SecretType } from '@console/internal/components/secrets/create-secret';
+import { SecondaryStatus } from '@console/shared';
+import { SecretKind } from '@console/internal/module/k8s';
+import './SecretsList.scss';
+
+type SecretsProps = {
+  secrets?: FirehoseResult<SecretKind[]>;
+};
+
+type SecretsListProps = {
+  namespace: string;
+};
+
+const secretTypes = [SecretType.dockerconfigjson, SecretType.basicAuth, SecretType.sshAuth];
+
+const Secrets: React.FC<SecretsProps> = ({ secrets }) => {
+  const filterData = _.filter(secrets.data, (secret) => _.includes(secretTypes, secret.type));
```

Comment:
Secrets are not listed in alphabetical order.
![image](https://user-images.githubusercontent.com/9964343/78133923-e8c05500-743c-11ea-91d9-f252a0ad54fe.png)

use `_.sortBy(unSortedData, ({ metadata: { name } }) => name.toLowerCase());` here in the array to sort it alphabetically.

Reply:
FWIW, you probably don't need to do `name.toLowerCase()` as you're using the metadata name and that's always kebab case.